Patches/PlayerControlPatch.cs

```diff
@@ -1159,6 +1159,64 @@ public static bool RpcCheckAndMurder(PlayerControl killer, PlayerControl target,
                     return false;
                 }
                 break;
+            //击杀萧暮
+            case CustomRoles.XiaoMu:
+                var Fg = IRandom.Instance;
+                int XiaoMu = Fg.Next(1, 4);
```

Comment:
IRandom.Instance.Next "Generates a random number greater than or equal to 0 and less than maxValue." - So in this case, 4 will never be picked. Make it 1 to 5.

Reply:
fix